specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/ga/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,932 @@
+        },
+        "nextLink": {
+          "description": "Represents the URL link to the next page of phone number results",
+          "type": "string"
+        }
+      }
+    },
+    "PhoneNumberLongRunningOperation": {
+      "description": "Represents the status of a long running operation",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Represents the status of a long running operation",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberLongRunningOperationStatusCode",
+            "modelAsString": true
+          }
+        },
+        "resourceLocation": {
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "error": {
+          "$ref": "#/definitions/CommunicationError"
+        },
+        "id": {
+          "type": "string"
+        },
+        "longRunningOperationType": {
+          "description": "Represents a category of long running operation",
+          "enum": [
+            "Purchase",
+            "ReleasePhoneNumber",
+            "Search",
+            "UpdatePhoneNumberCapabilities"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "LongRunningOperationType",
+            "modelAsString": true
+          }
+        },
+        "lastActionDateTime": {
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SearchRequest": {
+      "description": "Represents a phone number search request.",
+      "required": [
+        "assignmentType",
+        "capabilities",
+        "phoneNumberType"
+      ],
+      "type": "object",
+      "properties": {
+        "phoneNumberType": {
+          "description": "The phone number type of the search.",
+          "enum": [
+            "Geographic",
+            "TollFree",
+            "Unknown"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberType",
+            "modelAsString": true
+          }
+        },
+        "assignmentType": {
+          "description": "The assignment type of the search.",
+          "enum": [
+            "Unknown",
+            "User",
+            "Application"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberAssignmentType",
+            "modelAsString": true
+          }
+        },
+        "capabilities": {
+          "$ref": "#/definitions/PhoneNumberCapabilities"
+        },
+        "areaCode": {
+          "description": "The area code of the search.",
+          "type": "string"
+        },
+        "quantity": {
+          "format": "int32",
+          "description": "The quantity of phone numbers in the search.",
```

Comment:
This is a meta description, let's make it more actionable like "The desired quantity of phone numbers."

Reply:
Done.